src/framework/element.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Creates DOM Node. Implements jsx-parser's createElement API
+ * @param tag - HTML tag as string or Component function
+ * @param props - element properties as parsed by jsx-parser
+ * @param children - child elements
+ * @returns {DocumentFragment|Element}
+ */
+export const createElement = (tag, props, ...children) => {
+  if (typeof tag === 'function') return tag({ ...props, children }, children);
```

Comment:
Don't we require curly braces for `if` statements?

Reply:
Fixed code style.
Relied on linter. Trust but verify